src/views/OrderDetail.vue

```diff
@@ -0,0 +1,199 @@
+              </ion-item>
+              <ion-item class="border-top">
+                <ion-radio :disabled="checkPreOrderAvailability(item, 'BACKORDER')" slot="start" value="BACKORDER" />
+                <ion-label >{{ $t("Back Order") }}</ion-label>
+                <ion-note slot="end" :color="getEstimatedDeliveryDate(item.sku, 'BACKORDER') ? '' : 'warning'">{{ getEstimatedDeliveryDate(item.sku, "BACKORDER") ? getEstimatedDeliveryDate(item.sku, "BACKORDER") : $t("No shipping estimates") }}</ion-note>
+              </ion-item>
+            </ion-radio-group>
+            
+          </ion-card>
+        </main>
+        <div class="text-center center-align">
+          <ion-button @click="updateDraftOrder(order.line_items)">{{ $t("Save changes to order") }}</ion-button>
+        </div>
+      </div>
+    </ion-content>
+  </ion-page>
+</template>
+<script lang="ts">
+import {
+  IonButton,
+  IonCard,
+  IonCheckbox,
+  IonContent,
+  IonHeader,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonNote,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  IonBackButton,
+  IonRadio,
+  IonRadioGroup
+} from "@ionic/vue";
+import {
+  sendOutline,
+  swapVerticalOutline,
+  callOutline,
+  mailOutline,
+} from "ionicons/icons";
+import { defineComponent } from 'vue';
+import { mapGetters, useStore } from 'vuex';
+import { useRouter } from 'vue-router';
+import { DateTime } from 'luxon';
+
+export default defineComponent({
+  name: 'Home',
+  components: {
+    IonButton,
+    IonCard,
+    IonCheckbox,
+    IonContent,
+    IonHeader,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonNote,
+    IonPage,
+    IonTitle,
+    IonToolbar,
+    IonBackButton,
+    IonRadio,
+    IonRadioGroup
+  },
+  computed: {
+    ...mapGetters({
+      order: 'order/getDraftOrder',
+      orderId: 'order/getCurrentDraftOrderId',
+      configId: 'shop/getShopConfigId'
+    })
+  },
+  data(){
+    return {
+      checkPreorderItemAvailability: [] as any
+    }
+  },
+  async mounted(){
+    if(this.$route.query.id){
+      this.store.dispatch('order/setCurrentDraftOrderId', this.$route.query.id);
+    }
+    await this.store.dispatch('order/getDraftOrder', {id: this.orderId, configId: this.configId });
+    const productIds = await this.order.line_items.map((item: any) => {
+      return item.sku;
+    }).filter((id: any) => id);
+    this.checkPreorderItemAvailability = await this.store.dispatch('shop/checkPreorderItemAvailability', productIds);
+  },
+  methods: {
+    checkPreOrderAvailability(item: any, label: string){
+      const product = this.checkPreorderItemAvailability.find((product: any) => {
+        return product.sku == item.sku && product.label == label
+      })
+      return !product;
+    },
+    addProperty (item: any, event: any) {
+      const product = this.checkPreorderItemAvailability.find((product: any) => {
```

Comment:
As we only have a single line in the block we can directly return without using block and return statement.